src/db/SQLJSPackageDB.ts

```diff
@@ -117,9 +117,9 @@ export class SQLJSPackageDB implements PackageDB {
     this.initialized = false;
   }
 
-  async initialize() {
+  async initialize(initSql?: initSqlJs.SqlJsStatic) {
```

Comment:
This adds an _optional_ dependency on `sql.js` in our API interface -- which will probably be fine in most cases, but has _potential_ to cause issues if/when the caller is using a version of `sql.js` that is different than FPL's version of `sql.js` in significant ways.

Looking at your FSH Online branch, it seems the main difference is it calls `initSqlJs` w/ the `locateFile` option set in the passed in [SqljsConfig](https://sql.js.org/documentation/global.html#SqlJsConfig):
```js
await initSqlJs({ locateFile: () => workletURL });
```

I think it might be better / more compatible if we had our `intialize` function take an `SqlJsConfig` type as its argument or if we even defined our own `SQLJSPackageDBOptions` type that had a `locateFile` property. It's safer to pass that across the API boundary than to pass a whole `initSqlJs.SqlJsStatic` across the boundary.

Reply:
I added a `SQLJSPackageDBInitializeOptions` type and passed that into `initialize`. I wasn't sure if I should have the type reflect that this is an options specifically for the `initialize` function, but I can simplify the type name to `SQLJSPackageDBOptions` if you think that's fine.

Reply:
I think this is fine. If you were to name it `SQLJSPackageDBOptions`, then I'd say it should be a constructor argument that saves the `locateFile` function as a private member and uses it during `initiatlize()` (which would mean `initialize` doesn't need an argument anymore). I'm not sure one approach is better than the other, so I'm good with how you have it.